lib/artifact.js

```diff
@@ -142,10 +159,10 @@ module.exports = (bluebird, fs, logger, request, requestOptions, requestPromise,
          * @param {(string|WriteStream)} downloadLocation The local file system
          *                                              path or WriteStream to
          *                                              save the artifact to.
-         * @return {Promise.<string>}
+         * @return {(Promise.<string>|null)}
```

Comment:
Right. It was suggested to do `{Promise.<string|null>}` on the JSDoc issue to include Promise formatting. I've fixed. a221c93